.github/workflows/codeql-analysis.yml

```diff
@@ -8,7 +8,7 @@ name: "CodeQL"
 on:
   workflow_dispatch:
   push:
-    branches: [main, v1.8.x, v2.0.x, v2.1.x, v2.6.x, v3.0.x, v3.1.x, v4.0.x, v4.1.x, v4.2.x, v4.3.x, v4.4.x, v4.5.x, v4.6.x, v4.7.x, v5.0.x, v5.1.x, v5.2.x, v5.3.x, v5.4.x, v6.0.x, v6.1.x, v6.2.x, v6.3.x, v6.4.x, v6.5.x, v6.6.x, v6.7.x, v7.0.x, v7.1.x, v7.2.x]
+    branches: [main, v10.4.x, v11.1.x, v11.2.x, v11.3.x, v11.4.x, v11.5.x]
```

Comment:
I don't believe we even have the 'dot x' (`v11.5.x`) branches anymore - backports go immediately into release branch (`v11.5.2`) now.

I think we change this to `[main, 'v*.*.*']`?

Reply:
hmmmmmm - there's a cron schedule that means this runs once a week... i don't really wanna include *every* branch 🤔 maybe something like `[main, 'v9.*.*', 'v10.*.*', 'v11.*.*']`?

Reply:
I don't think that's how it works?

It's runs on main once a week (why? idk if that makes much sense fwiw), and also on pushes to those specific branches. If I understand GHA syntax correctly, It doesn't run once a week on every matching branch.